website/docs/r/organization_token.html.markdown

```diff
@@ -38,8 +38,33 @@ Basic usage:
 ```hcl
 resource "tfe_organization_token" "test" {
   organization = "my-org-name"
-  expired_at = "2051-04-11T23:15:59+00:00"
 }
+
+resource "time_rotating" "example" {
+  rotation_days = 30
+}
+
+resource "tfe_organization_token" "test" {
+  organization = data.tfe_organization.org.name
+  expired_at = time_rotating.example.id
+}
+```
+
+Generating the `expired_at` string using the date tool in unix systems (darwin):
+```
+date -Iseconds -v"+30d"
+```
+
+Generating the `expired_at` string using the date tool in unix systems (linux):
+```
+date -Iseconds -d"+30 days"
+```
+
+Generating the `expired_at` string using the `timeadd` Terraform function:
+```
+$ terraform console
+> timeadd(timestamp(), "720h")
+"2023-07-21T02:02:23Z"
```

Comment:
With the time_rotating example, I don't think all this detail is necessary anymore. Also, can you add the `expired_at` token below in the "Attributes Reference" with specific detail/example of the format? (both resources)

Reply:
Good call out, will do!